templates/debug.twig

```diff
@@ -74,13 +93,13 @@
                  ng-show="mtmDebug.contentTab != 'logs'">
                 <ul class="navbar" role="menu" style="padding: 0;">
                     <li class="menuTab" role="menuitem">
-                        <span class="item" style="font-weight: normal;"> Events</span>
+                        <span class="item" style="font-weight: normal;"> Events <br><br> <input type="checkbox" class="onlyFiredTags-chk" name="onlyfiredTags" value="1" ng-model="onlyfiredTags"> Only fired tags</span>
                     </li>
                     <li ng-show="(mtmDebug.mtmEvents|length) == 0">No event executed</li>
                     <li class="menuTab" role="menuitem" ng-class="{'active': eventIndex === mtmDebug.selectedEventIndex}"
-                    ng-repeat="event in mtmDebug.mtmEvents | orderBy:'$index':true">
+                    ng-repeat="event in mtmDebug.mtmEvents | orderBy:'$index':true" ng-hide="!event.tags.length && onlyfiredTags">
                         <a class="item ng-binding"  ng-click="mtmDebug.selectEvent(event.index0)" title="Time: {{ '{{ event.time }}'|raw }}. Trigger: '{{ '{{ event.metTrigger.name }}'|raw }}'">
-                            {{ '{{ event.index }}'|raw }}: {{ '{{ event.name }}'|raw }}
+                            {{ '{{ event.index }}'|raw }}: {{ '{{ event.name }}'|raw }} <span class="{{ '{{ event.tags.length ? \'icon-ok\' : \'icon-error\' }}' }}"></span>
```

Comment:
![Screenshot from 2022-01-25 12-42-13](https://user-images.githubusercontent.com/14902277/150928699-b45f1fdf-2be1-4127-8681-5b0cd70fd582.png)

@tsteur 
Using the existing icons from Matomo core there is 1 problem, if the Matomo instance is on https and the tracking website is on http the icon doesn't seems to load due to CORS.

Should we add a base64 encoded svg as an inline CSS to solve this ?

Reply:
sounds good 👍